nrf_security/src/mbedtls/CMakeLists.txt

```diff
@@ -133,11 +131,17 @@ if(TARGET ${IMAGE}mbedcrypto_cc310)
   target_link_libraries(${IMAGE}mbedcrypto_vanilla PRIVATE ${IMAGE}mbedcrypto_cc310)
 endif()
 
-# mbeddrbg library.
-# In order to ensure correct linking to glue functions from ctr_drbg this file
-# must be placed in independent library.
-zephyr_library_named(mbeddrbg)
+# mbedwrap library.
+# In order to ensure correct linking to glue functions from the following files
+# those must be placed in independent library to ensure they will always link
+# to the correct renamed symbols or glue layer.
+# In no symbols has been renamed they will link to correct implementation in
+# selected backend library.
+zephyr_library_named(mbedwrap)
 zephyr_library_sources(${ARM_MBEDTLS_ROOT}/library/ctr_drbg.c)
+zephyr_library_sources(${ARM_MBEDTLS_ROOT}/library/pk_wrap.c)
+zephyr_library_sources(${ARM_MBEDTLS_ROOT}/library/md_wrap.c)
+zephyr_library_sources(${NRF_MBEDTLS_ROOT}/src/mbedtls/replacements/cipher_wrap.c)
```

Comment:
Addition of CONFIG_MBEDTLS_CIPHER_AES_256_ECB_C, CONFIG_MBEDTLS_CIPHER_AES_256_CBC_C, CONFIG_MBEDTLS_CIPHER_AES_256_CTR_C and CONFIG_MBEDTLS_CIPHER_AES_256_CCM_C here as well?

Reply:
yep, fixed